### ISSUE TITLE
AttributeError: 'Message' object has no attribute 'message'

### ISSUE THREAD

**Issue by JeuDe** [blueset/efb-telegram-master#48]
My coding skill is poor,I spent two hours to try solving this,But I failed. 

The error message like this

```
ERROR]: telegram.ext.dispatcher (dispatcher.process_update; dispatcher.py:370)
    An uncaught error was raised while processing the update
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/telegram/ext/dispatcher.py", line 329, in process_update
    check = handler.check_update(update)
  File "/usr/local/lib/python3.6/dist-packages/efb_telegram_master/global_command_handler.py", line 34, in check_update
    res = self.filters(message)
  File "/usr/local/lib/python3.6/dist-packages/telegram/ext/filters.py", line 75, in __call__
    return self.filter(update)
  File "/usr/local/lib/python3.6/dist-packages/telegram/ext/filters.py", line 823, in filter
    return update.message is not None or update.edited_message is not None
AttributeError: 'Message' object has no attribute 'message'

```

Can you help me?
Thanks

**Comment by blueset** [comment]
Bug caused by improperly specified version constraint of dependency `python-telegram-bot`.

Fixed in version 2.0.0b20.

**Comment by Lzx1085325316** [comment]
我发现了很奇怪的解决方法：在出现错误的group里写/chat，回车，就能正常使用了。不过我的是在只link了一个人的group里操作的，其他的还没遇到
